packages/lit-virtualizer/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "inlineSources": true,
     // "outDir": "./development/",
     "outDir": "./",
-    "rootDir": "./src",
+    // "rootDir": "./src",
```

Comment:
Nit (no action required): Should this and `outDir` be commented vs removed entirely?

Reply:
I reverted this change after some difficulty getting things to behave.  I feel like a future PR might get rid of src subfolder and we just have sibling .d.ts/.js/.map files, but that would need a separate discussion.